backend/eventloop/workspace_event_loop.go

```diff
@@ -221,28 +221,32 @@ func workspaceEventLoopRouter(input chan workspaceEventLoopMessage, namespaceID
 
 	for {
 		wrapperEvent := <-input
-
 		event := (wrapperEvent.payload).(eventlooptypes.EventLoopMessage)
 
-		if wrapperEvent.messageType == workspaceEventLoopMessageType_Event {
-			// When the workspace event loop receive an event message, process it
+		processWorkspaceEventLoopMessage(ctx, event, wrapperEvent, state, log)
```

Comment:
Comment for reviewers: I moved the contents of the `workspaceEventLoopRouter` function into its own function (in order to make it easier to unit test), but the contents of the function is otherwise the same.